desktop/src/com/mygdx/game/client/ui/TextInputDialogFactory.java

```diff
@@ -0,0 +1,91 @@
+package com.mygdx.game.client.ui;
```

Comment:
Hmm jest okej ale jakoś bym to wkomponował w to co już mamy, ale to już ja zrobię